docs/site/Server.md

```diff
@@ -58,6 +58,37 @@ export class HelloWorldApp extends RestApplication {
 
 ## Configuration
 
+### Enable HTTPS
+
+Enabling HTTPS for the LoopBack REST server is just a matter of specifying the
+protocol as `https` and specifying the crendential details.
+
+In the following app we configure HTTPS for a bare miminum app using a key +
+certificate chain variant.
+
+```ts
+import {RestApplication, RestServer, RestBindings} from '@loopback/rest';
+import * as fs from 'fs';
+
+export async function main() {
+  const options = {
+    rest: {
+      protocol: 'https',
+      key: fs.readFileSync('./key.pem'),
+      cert: fs.readFileSync('./cert.pem'),
+    },
+  };
+  const app = new RestApplication(options);
+  app.handler(handler => {
```

Comment:
Can we remove the rest of the code below ... and add a comment `// original implementation`?

Reply:
Let's keep it. It makes for a good experience to have a completely working code example when one's learning about something, especially for beginners. Besides, it's not a lot of code; just enough to provide the whole context.

Reply:
@virkt25 thoughts?